docs/source/models/concept_models/vivarium_moud/concept_model.rst

```diff
@@ -0,0 +1,177 @@
+.. role:: underline
+    :class: underline
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _2021_concept_model_vivarium_moud:
+
+=====================================================
+Medications for Opioid Use Disorder (MOUD) Simulation
+=====================================================
+
+.. contents::
+  :local:
+  :depth: 1
+
+
+
+1 Overview
+++++++++++
+
+This document outlines the concept model for simulating Medications for Opioid Use Disorder (MOUD) interventions. The core of this model is a state-transition model representing opioid use disorder states and treatment dynamics.
+
+2 Modeling aims and objectives
+++++++++++++++++++++++++++++++
+
+TK - Objectives and goals of the MOUD modeling effort
```

Comment:
What does "TK" mean?

Reply:
"to come", intentionally misspelled as a journalism joke